### DISCUSSION TITLE
Change Class name of Window

### DISCUSSION THREAD

**Discussion by established94** [dotnet/wpf#8524] [General]
Hello Everyone, I was looking at my program's window using WinSpy++ and saw that the class name is:
HwndWrapper[ProjectName;;7b0784d0-5cd0-4f66-9c98-4c3664ee37ed] 

Is there any way to change this name?

**Comment by miloush** [comment]
No.

https://github.com/dotnet/wpf/blob/7fa7366ea20282a2254bc628ef02cf05a521ddef/src/Microsoft.DotNet.Wpf/src/Shared/MS/Win32/HwndWrapper.cs#L111-L112

Why do you need to change this name?

**Reply by established94** [reply]
No particular reason, I was just interested in modifying it.
